coverage_assessment.R

```diff
@@ -41,7 +41,118 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
```

Comment:
No need to repeat these lines of code as this is already in the top of the whole workflow file.

So, I would recommend deleting these lines of code

Reply:
Deleted.. Thanks!

Hi @ernestguevarra, I had done some other questions and committed and pushed from my R studio to github. However, I am not seeing the pull request in github?
What could be the reason?
Thanks!

Reply:
> Deleted.. Thanks!
> 
> Hi @ernestguevarra, I had done some other questions and committed and pushed from my R studio to github. However, I am not seeing the pull request in github? What could be the reason? Thanks!

Because this is the same branch, it will go to the same current pull request. I think I saw your changes already. Also, when you commit changes, make sure you tick the box of the file that you have made changes in (this is called staging). Make sure also that the file you are working on has been saved (the tab for the file will be red and with asterisk in the name if it hasn't been changed. Make sure all of this is done before pushing. once pushed the changes will be in the same pull request.